aws/resource_aws_eks_node_group.go

```diff
@@ -384,25 +385,25 @@ func resourceAwsEksNodeGroupRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("release_version", nodeGroup.ReleaseVersion)
 
 	if err := d.Set("remote_access", flattenEksRemoteAccessConfig(nodeGroup.RemoteAccess)); err != nil {
-		return fmt.Errorf("error setting remote_access: %w", err)
+		return diag.Errorf("error setting remote_access", err)
```

Comment:
Still need to pass `error`, just can't use `%w`.

```suggestion
		return diag.Errorf("error setting remote_access: %s", err)
```